docs/data/uci.py

```diff
@@ -0,0 +1,72 @@
+from io import BytesIO
+from typing import List, Tuple
+from tempfile import TemporaryDirectory
+import requests
+import zipfile
+
+import pandas as pd
+
+def get_census_dataset() -> Tuple[pd.DataFrame, pd.Series]:
```

Comment:
can we update these functions to download locally and re-use that local download? here's an example from sklearn: https://github.com/scikit-learn/scikit-learn/blob/e58f36c73/sklearn/datasets/_twenty_newsgroups.py#L152

Reply:
then we could move into the package proper